build.zig

```diff
@@ -7,20 +7,24 @@ const MODULE_NAME = "curl";
 
 pub fn build(b: *std.Build) void {
     const target = b.standardTargetOptions(.{});
+    const optimize = b.standardOptimizeOption(.{});
```

Comment:
Does this affect how libcurl is compile?

Reply:
I don't know about that, but I kept getting warnings from zig compiler when using zig-curl as dependency and passing my own optimize flags unless I did this.